Lib/ufo2ft/kernFeatureWriter.py

```diff
@@ -192,20 +238,73 @@ def _addGlyphClasses(self, lines):
         for key, members in sorted(self.groups.items()):
             lines.append("%s = [%s];" % (key, " ".join(members)))
 
-    def _addKerning(self, lines, kerning, enum=False, ignoreZero=False):
+    def _splitRtlKerning(self):
+        """Split RTL kerning into separate dictionaries."""
+
+        self.rtlGlyphPairKerning = {}
+        self.rtlLeftClassKerning = {}
+        self.rtlRightClassKerning = {}
+        self.rtlClassPairKerning = {}
+
+        classes = self._getClasses()
+        allKerning = (
+            (self.glyphPairKerning, self.rtlGlyphPairKerning, (False, False)),
+            (self.leftClassKerning, self.rtlLeftClassKerning, (True, False)),
+            (self.rightClassKerning, self.rtlRightClassKerning, (False, True)),
+            (self.classPairKerning, self.rtlClassPairKerning, (True, True)))
+
+        for origKerning, rtlKerning, (leftIsClass, rightIsClass) in allKerning:
+            for pair in list(origKerning.keys()):
+                lhs, rhs = pair
+                leftGlyphs = classes[lhs] if leftIsClass else [lhs]
+                rightGlyphs = classes[rhs] if rightIsClass else [rhs]
```

Comment:
lhs and rhs may be list string of glyphnames instead of classenames, if the classes went through self._liststr() due to removal of conflict glyphs.